packages/web/src/components/gcds-breadcrumbs/gcds-breadcrumbs-item.css

```diff
@@ -1,8 +1,7 @@
-:host .gcds-breadcrumbs-item {
+:host {
   position: relative;
   display: inline-block;
-  margin: var(--gcds-breadcrumbs-item-margin);
-  padding: var(--gcds-breadcrumbs-item-padding);
+  padding: var(--gcds-breadcrumbs-item-padding) !important;
```

Comment:
We can't remove the margin from the breadcrumb item, it's to create some spacing between the breadcrumb items on smaller screens.

Reply:
The bread crumb items are having some conflicts with ```* {
    -webkit-box-sizing: border-box;
    box-sizing: border-box;
    margin: 0;
    padding: 0;
}```
after the latest change. I am going to try and move the margin to the `a` tags instead to keep the spacing for mobile.